src/org/zaproxy/zap/extension/scripts/ZapAddOn.xml

```diff
@@ -23,7 +16,13 @@
 	<ascanrules/>
 	<pscanrules/>
 	<filters/>
-	<files/>
+	<files>
+		<file>scripts/scripts/extender/Copy as curl command menu.js</file>
```

Comment:
Is this to be a script or template? (If the former it needs to be manually added by the add-on?)